core-primitives/sgx/crypto/src/aes.rs

```diff
@@ -33,6 +32,9 @@ use std::{
 
 type AesOfb = Ofb<Aes128>;
 
+/// File name of the sealed AES key data.
+pub const AES_KEY_FILE_AND_INIT_V: &str = "aes_key_sealed.bin";
+
```

Comment:
[Nitpicking]

I think the filename should reflect that it contains the IV as too, not only the key. 

How about adding the `_data` suffix to the filename?

Reply:
Absolutely, good input! be3bc66f73072c994a8fd5f99bc9bda39a4214a5

However, this could in theory be a breaking change, but I doubt it affects anyone, which is why I did it after all, and marked the PR as breaking public API.